voting-machine/core/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "voting-machine-core"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+risc0-zkvm-core = { version = "0.7", default-features = false, features = ["pure"] }
+serde = { version = "1.0", default-features = false }
+
+[dev-dependencies]
+voting-machine-methods = { path = "../methods" }
+risc0-zkvm-serde = "0.7"
```

Comment:
`[dev-dependencies]` are meant to be used for tests, examples, and benchmarks. I didn't see any of those in this crate.

Reply:
Hmm yup looks like a bad copy/paste job. Good catch.